contrib/nosql/redis/redis.go

```diff
@@ -44,29 +45,28 @@ func init() {
 func New(config *gredis.Config) *Redis {
 	fillWithDefaultConfiguration(config)
 	opts := &redis.UniversalOptions{
-		Addrs:              gstr.SplitAndTrim(config.Address, ","),
-		Username:           config.User,
-		Password:           config.Pass,
-		DB:                 config.Db,
-		MaxRetries:         defaultMaxRetries,
-		PoolSize:           config.MaxActive,
-		MinIdleConns:       config.MinIdle,
-		MaxConnAge:         config.MaxConnLifetime,
-		IdleTimeout:        config.IdleTimeout,
-		PoolTimeout:        config.WaitTimeout,
-		DialTimeout:        config.DialTimeout,
-		ReadTimeout:        config.ReadTimeout,
-		WriteTimeout:       config.WriteTimeout,
-		IdleCheckFrequency: defaultIdleCheckFrequency,
```

Comment:
`IdleCheckFrequency`漏掉了，有替代配置项吗？

Reply:
官方删除了，使用了 ` MaxIdleConns ` 替代
```
## v9 UNRELEASED

- Added support for [RESP3](https://github.com/antirez/RESP3/blob/master/spec.md) protocol.
- Removed `Pipeline.Close` since there is no real need to explicitly manage pipeline resources.
  `Pipeline.Discard` is still available if you want to reset commands for some reason.
- Replaced `*redis.Z` with `redis.Z` since it is small enough to be passed as value.
- Renamed `MaxConnAge` to `ConnMaxLifetime`.
- Renamed `IdleTimeout` to `ConnMaxIdleTime`.
- Removed connection reaper in favor of `MaxIdleConns`.
- Removed `WithContext`.
```
[reference] https://github.com/redis/go-redis/commit/f6a8adc50cdaec30527f50d06468f9176ee674fe

Reply:
@houseme 那可以把不再使用的内部常量删除了哈，比如`defaultIdleCheckFrequency`